kogito-quarkus-examples/process-usertasks-timer-quarkus-with-console/docker-compose/docker-compose-postgresql.yml

```
@@ -63,7 +63,7 @@ services:
 
   keycloak:
     container_name: keycloak
-    image: quay.io/keycloak/keycloak:${KEYCLOAK_VERSION}-legacy
+    image: quay.io/keycloak/keycloak:${KEYCLOAK_VERSION}
```

@tiagodolphine I tested this image config and is not working locally, I think we need to continue using legacy for now.
```suggestion
    image: quay.io/keycloak/keycloak:legacy
```